crates/cli/src/subcommands/publish.rs

```diff
@@ -24,12 +25,12 @@ pub fn cli() -> clap::Command {
                 .help("The type of host that should be for hosting this module"),
         )
         .arg(
-            // TODO(jdetter): Rename this to --delete-tables (clear doesn't really implies the tables are being dropped)
-            Arg::new("clear_database")
-                .long("clear-database")
+            Arg::new("delete_tables")
+                .long("delete-tables")
                 .short('c')
```

Comment:
keeping the `-c` flag here kinda seems like its legacy. I believe we can't use `-d` because we're using that for the debug flag. Maybe we just keep `--clear-database` and drop the TODO, what do you think @bfops @cloutiertyler ?

Reply:
Tyler already responded on this PR - let's just change this back to `--clear-database` and this is good to merge, thanks Zeke 👍